usr/lib/systemd/system/greenboot-grub2-set-success.service

```diff
@@ -19,4 +19,4 @@ ExecStart=/usr/bin/grub2-editenv - set boot_success=1
 ExecStart=/usr/bin/grub2-editenv - unset boot_counter
 
 [Install]
-RequiredBy=multi-user.target
+WantedBy=multi-user.target
```

Comment:
please drop this change from this commit and instead cherry-pick the commit from https://github.com/fedora-iot/greenboot/pull/72 onto this branch (it has a commit message why this change is needed)

Reply:
It was cherry-picked but squashing commits removes other messages, so dropping it from here and rebasing from main where #72  which is already merged.